tests/Api/Admin/ProductOptionsTest.php

```diff
@@ -167,8 +167,10 @@ public function it_creates_a_product_option(): void
             ], \JSON_THROW_ON_ERROR),
         );
 
+        $response = $this->client->getResponse();
+
         $this->assertResponse(
-            $this->client->getResponse(),
+            $response,
```

Comment:
Since we use this variable only once, it was okay to inline the above code

Reply:
Lets stay consistent with the rest of code